src/index.ts

```diff
@@ -1,279 +1,164 @@
-import { ApiPromise, WsProvider } from "@polkadot/api";
-import { Header, Address } from "@polkadot/types/interfaces/runtime";
+import { Config, AppConfig } from "./config"
```

Comment:
I won't review this file much further until some of the things that we discussed elsewhere are done: 

1. the old config format, filtering and matching system should come back
2. with an additional filter of: instead of full-text search on the transaction data, search only the signer. 

Since this will probably be a revamp.